packages/client/src/lib.js

```diff
@@ -91,22 +91,26 @@ class Web3Storage {
    * @param {API.PutOptions} [options]
    * @returns {Promise<API.CIDString>}
    */
-  static async put({ endpoint, token }, files, { onStoredChunk, maxRetries = MAX_PUT_RETRIES } = {}) {
+  static async put({ endpoint, token }, files, { onCarCreated, onStoredChunk, maxRetries = MAX_PUT_RETRIES } = {}) {
```

Comment:
`onCarCreated` is accurate, but we could go with `onRootCidReady` or similar to keep the CARs hidden here... @alanshaw what say you?

Reply:
SGTM

Reply:
renamed 🎉